apis/v1alpha1/convert.go

```diff
@@ -175,3 +175,170 @@ func TargetAllocatorEmbedded(in OpenTelemetryTargetAllocator) v1beta1.TargetAllo
 	}
 	return out
 }
+
+func tov1alpha1(in v1beta1.OpenTelemetryCollector) (*OpenTelemetryCollector, error) {
```

Comment:
why do we need the conversion going this direction?

Reply:
It is mentioned in the PR description